mteb/tasks/Classification/san/SanskritShlokasClassification.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class SanskritShlokasClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="SanskritShlokasClassification",
+        description="This data set contains ~500 Shlokas  ",
+        reference="https://github.com/goru001/nlp-for-sanskrit",
+        dataset={
+            "path": "bpHigh/iNLTK_Sanskrit_Shlokas_Dataset",
+            "revision": "772c6ef188a4ffd2bd2e115fcefe5957f2a1dc1f",
+        },
+        type="Classification",
+        category="s2s",
+        date=("2014-01-01", "2018-01-01"),
+        eval_splits=["test"],
```

Comment:
There is no test split on HF, the only available splits are `train` and `validation`
```suggestion
        eval_splits=["train", "validation"],
```

Reply:
Done